src/index.html

```diff
@@ -3,11 +3,399 @@
+    <title>NAMU</title>
+    <link 
+      rel="icon" 
+      type="image/x-icon" 
+      href="./images/NAMU.svg"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@700&family=Raleway:wght@400;500;600&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+  <body class="page">
+    <header class="header">
+      <div class="top-bar">
+        <a class="logo-link" href="#">
+          <img 
+            class="logo" 
+            src="./images/logo.svg"
+            alt="logo"    
+          >
+        </a>
+        <div class="icons">
+          <a class="menu__link" href="#menu"></a>
+          <div class="lang">
+            <span>UA</span>
+            <img 
+              class="dropdown" 
+              src="./images/dropdown.svg"
+              alt="dropdown"  
+            >
+          </div>
+        </div>
+      </div>
+      <div class="header__content">
+        <div class="header__content-left">
+          <div class="intro">
+            <div class="intro__date">
+              <span>10 серпня - 10 листопада</span>
+            </div>
+            <div class="intro__header">Мистецтво ХІХ - ХХ ст.</div>
+            <div class="intro__sub">Внесок українських митців у світову культуру 19-20ст</div>
+            <a class="intro__link btn" href="#visit">Купити квиток</a>
+            <div class="intro__date-2">10.08 - 10.10</div>
+          </div>
+        </div>
+        <div class="header__content-right">
+          <div class="header__img"></div>
+        </div>
+      </div>
+    </header>
+
+    <aside id="menu" class="menu">
+      <div class="top-bar">
+        <img 
+          class="logo" 
+          src="./images/logo.svg"
+          alt="logo"  
+        >
+        <div class="icons">
+          <a class="close__link" href="#"></a>
+          <div class="lang">
+            <span>UA</span>
+            <img 
+              class="dropdown" 
+              src="./images/dropdown.svg"
+              alt="dropdown"  
+            >
+          </div>
+        </div>
+      </div>
+      <div class="menu__content">
+        <div class="menu__content-left">
+          <div class="menu__info">
+            <div class="menu__info--block">
+              <div class="menu__info--block-header">Розклад сьогодні:</div>
+              <div class="menu__info--block-sub">12:00 - 19:00</div>
+            </div>
+
+            <div class="menu__info--block">
+              <div class="menu__info--block-header">Адреса:</div>
+              <div class="menu__info--block-sub">
+                <a 
+                  class="address" 
+                  href="https://goo.gl/maps/B1m5RwzTnQz4H7P86" 
+                  target="_blank"
+                >Київ, вул. М. Грушевського, 6</a>
```

Comment:
Need to check and fix it everywhere
```suggestion
                <a 
                  class="address" 
                  href="https://goo.gl/maps/B1m5RwzTnQz4H7P86" 
                  target="_blank"
                >
                  Київ, вул. М. Грушевського, 6
                </a>
```